types/authorization.yaml

```diff
@@ -78,6 +85,21 @@ types:
           $ref: /iso15118_charger#/CertificateHashDataInfo
         minItems: 1
         maxItems: 4
+  TokenValidationStatusMessage:
+    description: The token validation status
+    type: object
+    additionalProperties: false
+    required:
+      - token
+      - status
+    properties:
```

Comment:
What do you think about adding an optional `connector` property to this type to be able to reference the connector/EvseManager that the authorization was provided to? The `TokenValidationStatusMessage` could be published twice in that case:

1. When the validation process finished (without `connector`)
2. As part notify_evse_callback . (with `connector`) in case the status is `Accepted`

Reply:
Do you think it's actually necessary? I think that the SessionStarted event from the EvseManager would be sufficient

Reply:
Yes you are right. With the SessionStarted(reason=Authorized) event you should have indeed everything you need